command/daemon/daemon.go

```diff
@@ -58,22 +57,22 @@ func (c *daemonCommand) run(*kingpin.ParseContext) error {
 	}
 
 	// load the configuration from the environment
-	config, err := fromEnviron()
+	env, err := fromEnviron()
 	if err != nil {
 		return err
 	}
 
-	// TODO: These can be set in the config struct as `required: "true"` instead
+	// TODO: These can be set in the env struct as `required: "true"` instead
 	// once we have separate configs for the daemon and the delegate
-	if config.Client.Host == "" {
+	if env.Client.Host == "" {
 		return errors.New("missing DRONE_RPC_HOST")
 	}
-	if config.Client.Secret == "" {
+	if env.Client.Secret == "" {
```

Comment:
these can be marked as `required` in the envconfig. see https://github.com/kelseyhightower/envconfig#struct-tag-support